src/pipeline_v3/cli/management.py

```
@@ -588,6 +609,103 @@ def _parse_index_type(self, index_type_str: str) -> IndexType:
+        
+        if not args.processing_options:
+            args.processing_options = ','.join(profile.processing_options)
+            logger.debug(f"Profile: Set processing_options to {args.processing_options}")
+        
+        # Apply timeout multiplier
+        if profile.timeout_multiplier != 1.0:
+            original_timeout = self.config.pipeline.timeout_seconds
+            self.config.pipeline.timeout_seconds = int(original_timeout * profile.timeout_multiplier)
+            logger.debug(f"Profile: Adjusted timeout from {original_timeout}s to {self.config.pipeline.timeout_seconds}s")
+    
+    def _parse_processing_options(self, options_str: Optional[str]) -> Dict[str, bool]:
```

Ruff (0.11.9)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)